src-electron/db/db-mapping.js

```diff
@@ -700,7 +701,8 @@ exports.map = {
       incoming: dbApi.fromDbBool(x.INCOMING),
       outgoing: dbApi.fromDbBool(x.OUTGOING),
       isIncoming: dbApi.fromDbBool(x.IS_INCOMING),
-      isEnabled: dbApi.fromDbBool(x.IS_ENABLED)
+      isEnabled: dbApi.fromDbBool(x.IS_ENABLED),
+      isLargeMessage: dbApi.fromDbBool(x.IS_LARGE_MESSAGE)
```

Comment:
endpointTypeCommand(any map which has a corresponding table) normally denotes to only the fields in the endpoint_type_command table. So if another table's entries are needed then create/use endpointTypeCommandExtended for it and add this there.

Reply:
I believe we have messed this up for a certain set of tables/maps but that was the idea behind the proper mapping.